src/VisualStudio/IntegrationTests/VisualBasic/BasicBuild.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Roslyn.VisualStudio.Test.Utilities;
+using Xunit;
+
+namespace Roslyn.VisualStudio.Integration.UnitTests
+{
+    [Collection(nameof(SharedIntegrationHost))]
+    public class BasicBuild
+    {
+        private IntegrationHost _host;
+
+        public BasicBuild(IntegrationHost host)
+        {
+            _host = host;
+            _host.Initialize();
```

Comment:
@jasonmalinowski. Calling `Initialize()` is required so that the host can be shared between multiple tests and it can properly connect to the host process and enable the remoting service, even if the previous instance died or had a failed test (and therefore a new host should be created).